src/parameterizations/vertical/MOM_energetic_PBL.F90

```diff
@@ -463,11 +472,12 @@ subroutine energetic_PBL(h_3d, u_3d, v_3d, tv, fluxes, visc, dt, Kd_int, G, GV,
                        ! layer [R Z3 T-3 ~> W m-2].
     diag_ustar_BBL, &  ! The bottom boundary layer friction velocity [H T-1 ~> m s-1 or kg m-2 s-1]
     diag_BBL_decay_scale, & ! The bottom boundary layer TKE decay length scale [H ~> m]
-
-    diag_mStar_MIX, &  ! Mstar used in EPBL [nondim]
-    diag_mStar_LT, &   ! Mstar due to Langmuir turbulence [nondim]
+    diag_mstar_sfc, &  ! mstar used in EPBL [nondim]
+    diag_mstar_BBL, &  ! mstar used in EPBL BBL [nondim]
+    diag_mstar_LT, &   ! mstar due to Langmuir turbulence [nondim]
     diag_LA, &         ! Langmuir number [nondim]
-    diag_LA_MOD        ! Modified Langmuir number [nondim]
+    diag_LA_mod, &     ! Modified Langmuir number [nondim]
+    diag_ustar, diag_bflx
```

Comment:
There need to be commends added here describing the real variables `diag_ustar` and `diag_bflx` and their units.